include/seqan3/core/detail/strong_type.hpp

```diff
@@ -390,6 +391,33 @@ class strong_type
     }
     //!\}
 
+    /*!\name Comparison operators.
+     * \brief Only available if the corresponding skill from seqan3::detail::strong_type_skill is added.
+     *
+     * \if DEV
+     * Implemented as member functions because requires does not work on friends.
+     * \endif
+     * \{
+     */
+    //!\brief Return whether this instance is equal to `other`.
```

Comment:
```suggestion
    //!\brief Return whether this instance is equal to `rhs`.
```
Very often we use the term lhs and rhs for the comparison operations. I would like to keep this as much as possible.